sdkcompat/v252/BUILD

```diff
@@ -34,6 +34,10 @@ java_library(
         ]),
         intellij_ue = glob([
             "com/google/idea/sdkcompat/javascript/**",  #api223
+            "com/google/idea/sdkcompat/scala/**",  #api250
```

Comment:
The api tag seems wrong here, please use <version where the api changed> - 1. So that if we drop the previous version we can just grep for the API tag.

Reply:
If I understand this correctly the API was introduced in 251, please use 243 as the API tag.